src/coherence/client.py

```diff
@@ -846,6 +857,22 @@ async def is_empty(self) -> bool:
         v = await self._client_stub.isEmpty(r)
         return self._request_factory.serializer.deserialize(v.value)
 
+    @_pre_call_cache
+    async def is_ready(self) -> bool:
+        try:
+            r = self._request_factory.is_ready_request()
+            v = await self._client_stub.isReady(r)
+            return self._request_factory.serializer.deserialize(v.value)
+        except grpc.aio._call.AioRpcError as e:
+            if e.code() == grpc.StatusCode.UNIMPLEMENTED:
+                raise OperationNotSupportedError(
+                    "This operation is not supported by the gRPC proxy for the connected Coherence Server."
+                    "Please upgrade to a version that supports this operation."
+                ) from e
+            raise  # Re-raise all other gRPC errors
+        except Exception as e:
```

Comment:
Maybe no catch the Exception and let that bubble up?  I don't think we do this for any other methods.